### PR TITLE
Refactor registry package to repo package

### PR DESCRIPTION
**Pull request by jackgr**
Create a serializable configuration for use by expansion engines per discussion w/@sparkprime.

### CODE REVIEW

File: pkg/chart/chart.go [hunk 1]
```diff
@@ -416,3 +425,22 @@ func (c *Chart) loadMember(filename string) (*ChartMember, error) {
 
 	return result, nil
 }
+
+type ChartContent struct {
+	Chartfile *Chartfile     `json:"chartfile"`
+	Members   []*ChartMember `json:"templates"`
```
Comment (sparkprime) [inline pkg/chart/chart.go]: json: annotation is out of date

**Comment by sparkprime** [comment]
LGTM

**Comment by jackgr** [comment]
/cc @sparkprime, @michelleN 

Sorry, @sparkprime, didn't see the LGTM in the first commit. PTAL the new ones. :0)

**Comment by jackgr** [comment]
Fixed json: annotation out of date.

**Comment by jackgr** [comment]
@michelleN I believe this refactoring is finished, meaning that I don't believe we need to bring any more files over from pkg/registry.

Of course, there's work to be done to make other things use this package, but that'll be the fun part. :0)

**Pull request merged**